src/carbonio-files-ui-common/views/components/ViewModeComponent.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useCallback, useMemo } from 'react';
+
+import { useReactiveVar } from '@apollo/client';
+import { IconButton, Tooltip } from '@zextras/carbonio-design-system';
+import { useTranslation } from 'react-i18next';
+import { DefaultTheme } from 'styled-components';
+
+import { viewModeVar } from '../../apollo/viewModeVar';
+import { VIEW_MODE } from '../../constants';
+
+export const ViewModeComponent = (): React.JSX.Element => {
+	const viewMode = useReactiveVar(viewModeVar);
+	const [t] = useTranslation();
+
+	const iconButtonIconProps = useMemo<keyof DefaultTheme['icons']>(
```

Comment:
We should use the singular, or just _icon_
```suggestion
	const iconButtonIconProp = useMemo<keyof DefaultTheme['icons']>(
```